common-data/amplitude-sync-choices.json

```diff
@@ -1,4 +1,5 @@
 [
-  ["USERS", "Users"],
+  ["USERS", "Users v1 (deprecated)"],
+  ["USERS_V2", "Users v2"],
```

Comment:
We're deprecating the original "USERS" type b/c we want the server to re-sync all our users.